Helper/Customer.php

```diff
@@ -274,7 +274,7 @@ public function getShippingAddressData()
             return 'false';
         }
 
-        if ($this->hasBillingAddress() == false) {
+        if ($this->hasShippingAddress() == false) {
             return 'false';
```

Comment:
Maybe add this all in one if and one return
`if ($this->isUserLoggedIn() == false || $this->hasShippingAddress() == false)`